src/panels/Menu/TopMenuBar/Menus/WindowsMenu.tsx

```diff
@@ -17,20 +17,37 @@ export function WindowsMenu() {
       {menuItems.map((itemConfig) => {
         const item = menuItemsData[itemConfig.id];
         return (
-          <Menu.Item
-            key={item.componentID}
-            leftSection={item.renderIcon?.(IconSize.xs)}
-            onClick={() => {
-              addWindow(item.content, {
-                title: item.title,
-                position: item.preferredPosition,
-                id: item.componentID,
-                floatPosition: item.floatPosition
-              });
-            }}
-          >
-            {item.title}
-          </Menu.Item>
+          <Group>
```

Comment:
How about reducing the cap here? Maybe either to `'xs'` or `5`?
```suggestion
          <Group gap={'xs'}>
```